docs/pages/keys.rst

```diff
@@ -0,0 +1,9 @@
+.. _nuts-discovery-keys:
+
+*********************
+Keys and certificates
+*********************
+
+The figure at https://docs.corda.net/network-map.html shows the CA structure of Corda nodes. The *Nuts Discovery Service* represents the *security zone* in that figure. To startup the service, a few keys need to be generated and configured. More info on this can be found in :ref:`nuts-discovery-setup`. Only the root CA is configured at each node as trusted anchor. Both the Doorman and Network Map CA are trusted because they are signed by the root. The CertRole for each CA is important, to prevent mistakes configuration files for generating these certificates can be found in this repository.
```

Comment:
This does not link to a figure.
Also, the Corda docs are still pointing to v3. Should we point to Corda docs head (v4)?

Reply:
Problem with linking to their docs is that it'll break when they change their docs...